src/Components/ArticlePage.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from "react";
 import {useParams } from "react-router-dom";
 import {fetchArticleById} from '../api';
 import moment from 'moment';
```

Comment:
moment is a legacy package in maintenance mode, and recommends you use alternatives